py3/pyxtension/tests/test_Streams.py

```diff
@@ -733,6 +732,16 @@ def test_mpfastmap_time(self):
         self.assertSetEqual(res, expected)
         self.assertLessEqual(dt, 4)
 
+    def test_on_next_withOutTerminationFunction(self):
+        list_to_append_mock = Mock()
+        stream([1,2,3,4,5]).on_next(list_to_append_mock)
+        self.assertEqual(list_to_append_mock.call_count, 0)
```

Comment:
This test doesn't test what elements are sent to the `list_to_append_mock`, and if those would be None - this test still will pass, so we better test what elements are passed to list_to_append_mock, and in what order